packages/workspace/src/workspace/errors.ts

```diff
@@ -19,6 +19,7 @@ import { SaltoError } from '@salto-io/adapter-api'
 import { MergeError } from '../merger'
 import { ValidationError } from '../validator'
 import { ParseError } from '../parser'
+import {MAX_ENV_NAME_LEN} from "./workspace";
```

Comment:
We use single quotes as a convention
```suggestion
import {MAX_ENV_NAME_LEN} from './workspace';
```

Reply:
Also this introduces a circular dependency. May I suggest passing the const to the Error in the constructor?

Reply:
V, moved the const to the errors file